config/puma_prod.rb

```diff
@@ -18,9 +18,5 @@
 after_worker_boot do
   require 'prometheus_exporter/instrumentation'
   PrometheusExporter::Instrumentation::Puma.start
-end
-
-after_fork do
-  require 'prometheus_exporter/instrumentation'
```

Comment:
Are we only not doing this `after_fork` because we're using 1 process at the moment?